help/audacity.appdata.xml

```diff
@@ -2,7 +2,7 @@
 <component type="desktop">
   <id>org.audacityteam.Audacity</id>
   <launchable type="desktop-id">audacity.desktop</launchable>
-  <project_license>GPL-2.0 and CC-BY-3.0</project_license>
```

Comment:
Is the documentation packaged with this software?

Reply:
Yes, it is packaged with the binaries.

Reply:
> http://github.com/audacity/audacity/pull/798#discussion_r721917366:
> Yes, it is packaged with the binaries.

Thus, why was this merged?

Reply:
Because https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license says: 

>  It is not recommended to include the license for accompanying documentation (for example) in project_license